ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js

```diff
@@ -0,0 +1,265 @@
+                setShowMore(true);
+              }}
+            >
+              {t('ShowMore')}
+            </ButtonLink>
+          </Box>
+        )}
+      </Box>
+    );
+  };
+
+  const connectRequest = connectRequests ? connectRequests[0] : undefined;
+
+  if (!connectRequest) {
+    history.push(mostRecentOverviewPage);
+    return null;
+  }
+
+  trackEvent({
+    category: 'MMI',
+    event: 'Custodian onboarding',
+    properties: {
+      actions: 'Custodian RPC request',
+      custodian: connectRequest.custodian,
+      apiUrl: connectRequest.apiUrl,
+    },
+  });
+
+  let custodianLabel = '';
+
+  if (
+    connectRequest.labels &&
+    connectRequest.labels.some((label) => label.key === 'service')
+  ) {
+    custodianLabel = connectRequest.labels.find(
+      (label) => label.key === 'service',
+    ).value;
+  }
+
+  return (
+    <Box className="page-container">
+      <Box className="page-container__header">
+        <Text className="page-container__title">{t('custodianAccount')}</Text>
+        <Text className="page-container__subtitle">
+          {t('mmiAddToken', [connectRequest.origin])}
+        </Text>
+      </Box>
+      <Box padding={4} className="page-container__content">
+        {custodianLabel && (
+          <>
+            <Text padding={4} color={TextColor.textDefault}>
+              {t('custodian')}
+            </Text>
+            <Label
+              marginRight={4}
+              marginLeft={4}
+              color={TextColor.textAlternative}
+              className="add_custodian_token_confirm__url"
+            >
+              {custodianLabel}
+            </Label>
+          </>
+        )}
+
+        <Text padding={4} color={TextColor.textDefault}>
+          {t('token')}
+        </Text>
+        <Box
+          marginRight={4}
+          marginLeft={4}
+          className="add_custodian_token_confirm__token"
+        >
+          {renderSelectedToken()}
+        </Box>
+        {connectRequest.apiUrl && (
+          <Box>
+            <Text padding={4} color={TextColor.textDefault}>
+              {t('apiUrl')}
+            </Text>
+            <Text
+              marginRight={4}
+              marginLeft={4}
+              color={TextColor.textAlternative}
+              fontSize="14"
+              className="add_custodian_token_confirm__url"
+            >
+              {connectRequest.apiUrl}
+            </Text>
+          </Box>
+        )}
+      </Box>
+
+      {!complianceActivated && (
+        <Box marginTop={4} data-testid="connect-custodian-token-error">
+          <Text data-testid="error-message" textAlign={TEXT_ALIGN.CENTER}>
+            {connectError}
+          </Text>
+        </Box>
+      )}
+
+      <Box padding={4} className="page-container__footer">
+        {isLoading ? (
+          <footer>
+            <PulseLoader />
+          </footer>
+        ) : (
+          <footer>
+            <Button
+              type="default"
+              large
```

Comment:
If you use `Button` from `import { Button, BUTTON_SIZES, BUTTON_TYPES } from 'component-library'` you can do this
```suggestion
            <Button
              type={BUTTON_TYPES.SECONDARY}
              size={BUTTON_SIZES.LG}
```